src/rules/use-tokens/use-tokens.js

```diff
@@ -107,56 +59,50 @@ const tokensByCategory = category =>
     }))
     .value();
 
-const COLORS = tokensByCategory('colors');
-const SPACING_NAMES = new Set(tokensByCategory('spacings').map(x => x.name));
+const tokensByType = type =>
+  _.chain(TOKENS)
+    .filter({ type: type })
+    .map(({ name, value, deprecated }) => ({
+      name: _.camelCase(name),
+      value,
+      deprecated,
+    }))
+    .value();
+
+const COLORS = tokensByType('color').filter(function(el) {
+  return el.deprecated !== true;
+});
+
 const RADII_NAMES = new Set(tokensByCategory('radii').map(x => x.name));
 const BORDER_NAMES = new Set(tokensByCategory('borders').map(x => x.name));
 
 const checkColor = (node, context) => {
   const { key, value } = node;
   const isColor = COLOR_PROPS.includes(key.name) && value.type === 'Literal';
-
-  if (!isColor) {
+  const allowedColor = COLOR_ALLOWLIST.filter(c => value.value === c).length;
+  // We check if the colour is in the allowlist, or if it's a valid Backpack token
+  if (!isColor || allowedColor) {
     return;
   }
 
   const color = tinycolor(value.value);
 
-  const expectedToken = _.find(COLORS, { value: color.toRgbString() });
-
-  if (expectedToken) {
+  const matchedTokens = _.filter(COLORS, { value: color.toRgbString() });
+  console.log('matchedTokens', matchedTokens[0]);
```

Comment:
Do we need this log statement?

Reply:
Ah good catch! That was left over from debugging